incubator/sparkoperator/templates/spark-operator-with-metrics.yaml

```diff
@@ -0,0 +1,42 @@
+{{ if .Values.enableMetrics }}
```

Comment:
What if I want to use both webhooks and metrics?

Reply:
Yeah good point. In the README, I added a line saying the chart only supports metrics or webhooks, but not both. But that's probably not the right thing to do. What should the correct behaviors be if both are enabled?

Reply:
I think we should support both. We just need to make sure the service declares the ports for both. We only need one service for both. Is it possible to combine all there yamls into one?

Reply:
It should be possible. Also to properly initialize namespaces. You might have a look how different project do it.

Reply:
Ok I'll see what I can do by early next week.